pkg/controller/controller_utils.go

```diff
@@ -65,6 +65,8 @@ const (
 	// 500 pods. Just creation is limited to 20qps, and watching happens with ~10-30s
 	// latency/pod at the scale of 3000 pods over 100 nodes.
 	ExpectationsTimeout = 5 * time.Minute
+	//
```

Comment:
missing comment text, please describe the field and the impact of increasing the value.